src/Hl7.FhirPath/FhirPath/Expressions/ExpressionNode.cs

```diff
@@ -112,13 +112,13 @@ public FunctionCallExpression(Expression focus, string name, TypeInfo type, IEnu
             if (String.IsNullOrEmpty(name)) throw Error.ArgumentNull("name");
             Focus = focus;
             FunctionName = name;
-            Arguments = arguments ?? throw Error.ArgumentNull("arguments");
+            Arguments = arguments != null ? arguments.ToArray() : throw Error.ArgumentNull("arguments");
         }
 
         public Expression Focus { get; private set; }
         public string FunctionName { get; private set; }
 
-        public IEnumerable<Expression> Arguments { get; private set; }
+        public Expression[] Arguments { get; private set; }
```

Comment:
Is this needed? Because this is a breaking change: the signature of the property `Arguments` has changed

Reply:
Changed it back. It works as we discussed.